test/unit/specs/stores/search-store.spec.js

```diff
@@ -382,7 +382,7 @@ describe('Search Store', () => {
     it.each`
       searchType   | nextSearchType | expectedFilterCount
       ${AUDIO}     | ${IMAGE}       | ${25}
-      ${IMAGE}     | ${ALL_MEDIA}   | ${37}
+      ${IMAGE}     | ${ALL_MEDIA}   | ${12}
```

Comment:
Why this change?

Reply:
Normally, `All Content` view only has 12 filters available (License, License Types, Search By Creator), and the tests should show 12 filters. But in tests, for some reasons, the watcher was probably not firing, and thus it was showing 37 filters on 'All Content' view.
I've added the tests starting on lines 412 to make sure that we do fix this problem in the future.
After the conversion of the search store to options store, the filters are changed correctly, so I removed the now redundant tests. The `image` -> `all_media` change that is tested on line 385 shows that the correct number of filters is shown.

Note: I did not see this incorrect number of filters in the live app, only in tests.

Reply:
I see! Thanks for explaining.